packages/cli/test/services/package-manager/rush.test.ts

```diff
@@ -10,6 +10,12 @@ import { PackageManagerService } from '../../../src/services/package-manager'
 const TestFileSystem = makeTestFileSystem()
 const TestProcess = makeTestProcess()
 
+const mapEffError = <R, A>(effect: Effect<R, { error: Error }, A>) =>
+  pipe(
+    effect,
+    mapError((e) => e.error)
+  )
```

Comment:
This is being used in many places, should we declare it on a separate file and export it?

Reply:
Probably, but given its for tests right now and I didn't really wanted to make it explicit I decided to leave it like this, thanks!